tests/security/static/README.md

```diff
@@ -0,0 +1,25 @@
+# CodeQL Static Analysis on OpenEnclave
+
+## CodeQL Introduction
+CodeQL is the static analysis engine used by developers to automate security checks, and by security researchers to perform variant analysis.
+
+Unlike traditional static analysis tools CodeQL provides an ability to extend the analysis via custom queries in addition to the built-in queries.
+
+## Running CodeQL analysis on OpenEnclave
+1. Run scripts/init.sh to download CodeQL library repoistory and CodeQL CLI.
+2. Run scripts/run.sh which builds the CodeQL database and executes OpenEnclave custom queries on it. At the end of the scan it generates the results in SARIF format (OpenEnclave.sarif).
+3. Install [CodeQL extension for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=GitHub.vscode-codeql) or [SARIF viewer extension](https://marketplace.visualstudio.com/items?itemName=WDGIS.MicrosoftSarifViewer).
```

Comment:
Is the CodeQL extension required for running these queries?

Reply:
> Is the CodeQL extension required for running these queries?

CodeQL extension/SARIF viewer extension is only required to run a query in VS code or to view results in VSCode. The scan itself doesn't require VS extensions. CLI is independent of VSCode extension.

Reply:
This is the extension of this PR and regarding CI integration but not blocking on this addition. When this is run by CLI, do you get a "FAILED" or "PASS" or is it more of a notification?

If you would like to determine this something that can be done as part of a separate PR (to not block the work here) is a bit of a hack but to add the scripts to run here: https://github.com/openenclave/openenclave/blob/master/.jenkins/pipelines/Azure/Linux/Jenkinsfile#L71

Reply:
The scripts in this PR will produce OpenEnclave.sarif as result, but it will not return a FAILED/PASS status. We could implement a python script to validate the results by parsing the sarif file. Perhaps in a new PR :)

The scripts I added here is mainly for running local CodeQL scanning for developers. Once we agree up on a final CI/CD approach for codeqL scan (Jenkins vs Github code scan action), We could write PR validation scripts accordingly.